src/patternfly/components/OptionsMenu/options-menu.scss

```diff
@@ -294,6 +294,10 @@
   & + & {
     padding-top: var(--pf-c-options-menu__group--group--PaddingTop);
   }
+
+  .pf-c-options-menu__separator:last-child {
```

Comment:
can you nest this within the existing `.pf-c-options-menu__separator` selector using `:last-child` or does that break things? That that it's a requirement, but that's how the dropdown and app launcher are written, which should have the same layout.

Reply:
Doh, yes that's how it should be.